usr/src/man/man1m/zfs.1m

```diff
@@ -1586,13 +1612,19 @@ option to the
 command, or by changing the reservation after the volume has been created.
 A
 .Qq sparse volume
-is a volume where the reservation is less then the volume size.
+is a volume where the reservation is less than the size of the volume plus the
+space required to store its metadata.
 Consequently, writes to a sparse volume can fail with
 .Er ENOSPC
 when the pool is low on space.
 For a sparse volume, changes to
 .Sy volsize
 are not reflected in the reservation.
+A volume that is not sparse is said to be
+.Qq thick provisioned .
+A sparse volume can become thick provisioned
+by setting the reservation to
```

Comment:
I think this should say `refreservation`.  You'd only use the reservation on ancient-version pools (as you noted elsewhere).

Reply:
Yeah, I was unsure about that.  Other places mention reservation in a generic context (without `.Sy`) to refer to either.  I think that changing to `refreservation` will cause less confusion, so will go with that.